src/PointEval/point_values.jl

```diff
@@ -24,17 +24,23 @@ struct PointValues{CV} <: AbstractValues
     PointValues{CV}(cv::CV) where {CV} = new{CV}(cv)
 end
 
-PointValues(cv::CellValues) = PointValues(eltype(cv.M), cv.ip, cv.gip)
-function PointValues(ip::Interpolation, ipg::Interpolation = default_geometric_interpolation(ip))
-    return PointValues(Float64, ip, ipg)
+function PointValues(cv::CellValues)
+    T = typeof(getdetJdV(cv, 1))
+    ip_fun = get_function_interpolation(cv)
+    ip_geo = get_geometric_interpolation(cv)
+    update_gradients = get_function_difforder(cv) == 1
```

Comment:
Here is one example where it would be nice if the flags were stored I guess.

Reply:
Not sure - only `DiffOrder` should currently be forwarded though, `update_detJdV` is always false for `PointValues`. From that POV a constructor taking `DiffOrder` would be nice :)